static/css/introjs.scss

```diff
@@ -0,0 +1,347 @@
+.introjs-overlay {
```

Comment:
I think it would be better if these were in `static/vendor/introjs/...`

Also, we should copy this over as CSS and not do SCSS on it (assuming it is actually just plain CSS)